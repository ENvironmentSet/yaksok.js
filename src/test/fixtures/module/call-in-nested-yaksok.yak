약속 밖
    약속 안
        @b b 출력
    안
밖
