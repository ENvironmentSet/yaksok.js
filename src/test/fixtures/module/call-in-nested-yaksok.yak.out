부수효과
b
